### ISSUE TITLE
BUG: Click on "Create New" does not trigger input validation in `CreationDialog`

### ISSUE THREAD

**Issue by grebaldi** [neos/neos-ui#3319]
### Description

Given a node type that configures several node creation dialog inputs with validation, said validation is not triggered when the "Create New" button of the creation dialog is clicked. Instead, the validation only gets triggered if at least one of the inputs receives at least one input event.

### Steps to Reproduce

Here's a node type that should enable reproduction:
```yaml
'Vendor.Site:Content.Example':
  superTypes:
    'Neos.Neos:Content': true

  ui:
    label: 'Validation example'
    inspector:
      groups:
        examples:
          label: 'Examples'

  properties:
    example1:
      type: string
      ui:
        label: 'Example #1'
        showInCreationDialog: true
        reloadIfChanged: true
        inspector:
          group: examples
      validation:
        'Neos.Neos/Validation/NotEmptyValidator': []
    example2:
      type: string
      ui:
        label: 'Example #2'
        showInCreationDialog: true
        reloadIfChanged: true
        inspector:
          group: examples
      validation:
        'Neos.Neos/Validation/NotEmptyValidator': []
    example3:
      type: string
      ui:
        label: 'Example #3'
        showInCreationDialog: true
        reloadIfChanged: true
        inspector:
          group: examples
```

1. Create a node with a node type like the one described above
2. Click on "Create New" in the node creation dialog
3. See nothing happen

As soon as you type something into any of the input fields, validation gets triggered immediately. This can be irritating however, since nothing tells the user that any of those fields are required. The only thing you see is that clicking on "Create New" apparently doesn't work.

#### Expected behavior

A click on "Create New" triggers the input validation, after which the button should disable itself (given that validation errors occurred).

#### Actual behavior

https://user-images.githubusercontent.com/2522299/210357939-e3c568da-76aa-4a1a-9b65-548279197efa.mp4

### Affected Versions

Neos: 7.3

UI: 7.3
